### DISCUSSION TITLE
Team reserve function

### DISCUSSION THREAD

**Discussion by FUCryptoClub** [hashlips-lab/nft-erc721-collection#392] [Q&A] [answered]
I was wondering if there's a possibility to add a 'team reserve' to the contract?

For example you have a 10,000 NFT collection and you want to reserve 100 to your wallet for minting. So if you do a public sale at the cost of 0.1ETH per mint, the contract will only allow 9,900 NFTs to be minted, leaving 100 NFTs for the 'team reserve' to be minted at any time for free (plug gas obviously).

I know there is a work around for this and thats to open the whitelist with only your wallet address in the list, mint the 100 and then close the whitelist or then update the contract and add the other whitelist addresses (if any) etc.

The reason for this idea, is because someone may not be financially capable of minting 100 on ethereum before they have made a sale on minting.

Thanks in advance.

**Comment by liarco** [comment, marked as answer]
Hi, I made a video where I show how to do this: https://youtu.be/edYfMHnsI4I

I hope this helps 😉

**Reply by FUCryptoClub** [reply]
Awesome. I tried to find an answer before asking the question, you are a legend. Thank you.
